lang/en/tool_objectfs.php

```diff
@@ -141,8 +144,21 @@
 $string['settings:presignedurl:presignedminfilesize'] = 'Minimum size for Pre-Signed URL (KB)';
 $string['settings:presignedurl:presignedminfilesize_help'] = 'Minimum file size to be redirected to Pre-Signed URL.';
 
+$string['settings:presignedcloudfronturl:header'] = 'CLOUDFRONT Settings';
+$string['settings:presignedcloudfronturl:warning'] = 'Before enabling CLOUDFRONT Pre-Signed URL, please, make sure that all tests are passed successfully: ';
+$string['settings:presignedcloudfronturl:enablepresignedcloudfronturls'] = 'Enable CLOUDFRONT Pre-Signed URLs';
+$string['settings:presignedcloudfronturl:enablepresignedcloudfronturls_help'] = 'Enable CLOUDFRONT Pre-Signed URLs by setting up a Cloudfront Distribution profile at AWS.';
+$string['settings:presignedcloudfronturl:cloudfront_resource_domain'] = 'DOMAIN (inc. https://)';
+$string['settings:presignedcloudfronturl:cloudfront_resource_domain_help'] = 'Enter the domain name from which resources are requested at Cloudfront (refer to AWS Cloudfront Distribution)';
+$string['settings:presignedcloudfronturl:cloudfront_key_pair_id'] = 'Key_Pair ID from AWS';
+$string['settings:presignedcloudfronturl:cloudfront_key_pair_id_help'] = 'This is generated using AWS account \'root\' user (along with the private key .pem file).';
+$string['settings:presignedcloudfronturl:cloudfront_private_key_pem_file_pathname'] = 'PRIVATE Key .pem pathname';
+$string['settings:presignedcloudfronturl:cloudfront_private_key_pem_file_pathname_help'] = 'Full path and filename to the .pem file. Example: ' . $CFG->dataroot . '/ (CFG->dataroot) should be prepended to .pem filename. The actual .pem file must be available for file_get_content().';
```

Comment:
So you couldn't have known this, but this will cause a translation issue. Moodle has a way to pass in vars to lang strings eg:

$string['accesskey'] = 'Access key, {$a}';

https://docs.moodle.org/dev/String_API#Run-time_parameters

Also I would strong suggest to NOT prepend the dataroute. In fact I would go one step further and say that this must be at a fixed path eg:
/sitedata/objectfs/cloudfront.pem

Then there is one less setting and one less thing to go wrong.